README.md

```diff
@@ -20,10 +20,18 @@ Store leverages multiple request throttling to prevent excessive calls to the ne
 
 ### How to include in your project
 
-###### Include gradle dependency (as well as MavenCentral Snapshot repo)
+Artifacts are hosted on **Maven Central**.
+
+###### Latest version:
 
 ```groovy
-implementation 'com.dropbox.mobile.store:store4:4.0.0-alpha01'
+def store_version = "4.0.0-alpha01"
```

Comment:
how about also adding a maven-badge at the top of the file:
```
[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.dropbox.mobile.store/store4/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.dropbox.mobile.store/store4/)
```
see https://blog.softwaremill.com/maven-badges-the-app-5c1bdcceca29 for reference